Userland/Libraries/LibC/stdlib.cpp

```diff
@@ -696,6 +699,26 @@ int atoi(const char* str)
     return value;
 }
 
+// non standard function
+char* itoa(int value, char* buffer, int base)
```

Comment:
glibc has `_itoa`, and musl has a static helper function for this exact operation. I'm not strongly against this, but we should check if we break some ports by defining a non-standard function in LibC.

Reply:
How can I check this? We don't lint ports (as the build part will take forever).